pcsx2/GS.cpp

```diff
@@ -202,6 +217,7 @@ void __fastcall gsWrite64_generic( u32 mem, const mem64_t* value )
 void __fastcall gsWrite64_page_00( u32 mem, const mem64_t* value )
 {
 	gsWrite64_generic( mem, value );
+	GetColorBurst(mem, static_cast<u32>(value[0]));
```

Comment:
I would prefer to have the mem check outside of the function
```
if (mem == GS_SMODE1)
    GetColorBurst(.......)
```
It avoid a parameter.

And I don't like the cast here. `[0]` kinds of imply an array where it is a plain pointer. So a `*` might be better. That being said, maybe you could just use a 64 bits value for the gs register.